content/en/docs/collector/trace-receiver.md

```diff
@@ -1867,13 +1828,13 @@ Here is what `appendAtmSystemInstrScopeSpans` looks like after the update:
 }
 ```
 
-You can now update the `generateTraces()` function and add variables to
+You can now update the `generateTraces` function and add variables to
```

Comment:
```suggestion
You can now update the `generateTraces()` function and add variables to
```

Reply:
this conflicts with the other review above, which requested me to completely remove the parenthesis, whereas my original change was from `generateTraces()` to `generateTraces(numberOfTraces int)`. I will leave it as is for now.